### PR TITLE
Logo was not displayed because the css structure did not match html

### PR DESCRIPTION
**Pull request by itirkarp**
/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/OFFAPPS-341

### Risks
 - None

### CODE REVIEW

File: templates/layout.hdbs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <div class="sugar_app crm_integration_app">
   <div>
-    <h3>{{setting "name"}} <span class="loader" style="display: none;"></span> <span class="logo"></span></h3>
+    <header>
+      <span class="logo"></span>
+      <h3>{{setting "name"}} <span class="loader" style="display: none;"></span> </h3>
```
Comment (princemaple) [inline templates/layout.hdbs]: is the loader used? could we remove it if it's not?

**Comment by princemaple** [comment]
:+1:

File: templates/layout.hdbs (hunk 1 above)
Comment (itirkarp) [inline templates/layout.hdbs]: I'm not sure if it is used anywhere or not. Do you know why it was put in there?

Comment (princemaple) [inline templates/layout.hdbs]: Nope, can remove it if it's never mentioned in the js.

Comment (itirkarp) [inline templates/layout.hdbs]: @princemaple gone

**Comment by svizzari** [comment]
:+1:

**Pull request merged**